src/Packages/Passport/Runtime/Scripts/Private/PassportImpl.cs

```diff
@@ -36,7 +40,17 @@ public PassportImpl(IBrowserCommunicationsManager communicationsManager)
         public async UniTask Init(string clientId, string environment, string? redirectUri = null, string? deeplink = null)
         {
             this.redirectUri = redirectUri;
-            InitRequest request = new() { ClientId = clientId, Environment = environment, RedirectUri = redirectUri };
+
+            var versionInfo = new Dictionary<string, string>
```

Comment:
Should we just create a class for this version info? Rather than a dictionary?

Reply:
Would that be preferable over a Dictionary?

Reply:
Personally, I prefer it more than magic strings. Or at least define the keys as a constant at the top of the class.

Reply:
Moved the version info stuff to a separate class